hook-common/src/webhook.rs

```diff
@@ -139,12 +139,10 @@ pub struct WebhookJobMetadata {
 
 /// An error originating during a Webhook Job invocation.
 /// This is to be serialized to be stored as an error whenever retrying or failing a webhook job.
-#[derive(Serialize, Debug)]
+#[derive(Deserialize, Serialize, Debug)]
 pub struct WebhookJobError {
     pub r#type: app_metrics::ErrorType,
     pub details: app_metrics::ErrorDetails,
-    #[serde(serialize_with = "serialize_uuid")]
-    pub uuid: uuid::Uuid,
```

Comment:
I dropped this because it makes `failed` rows unnecessarily unique which hurts the `GROUP BY` aggregation I do. I don't think we get any special value of these `UUID`s and so I'd rather generate them when I create the `AppMetric`.

Reply:
Yup, I asked this question in my PR, happy to see it dropped, thank you :+1: